h/herizo.md

```diff
@@ -0,0 +1,7 @@
+# Herizo
+
+![Herizo](herizo.jpg)
+
```

Comment:
It looks good. Isn't there any information about him? I mean the biography.
If there isn't I think it is already good.